FakeServer/Program.cs

```diff
@@ -122,15 +123,31 @@ private static int Run(string[] args, Dictionary<string, string> initialData)
                     app.RemainingArguments.Add(optionsUrls.Value());
                 }
 
+                if (optionInit.HasValue()) {
+                    app.RemainingArguments.Add("--init");
+                    app.RemainingArguments.Add(optionInit.Value());
+                    var baseAppSettingsFile = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "appsettings.json");
+                    var newAppSettingFile = Path.Combine(initialData["currentPath"],"appsettings.json");
+                    if (!File.Exists(newAppSettingFile)) {
```

Comment:
If file already exiists, write to console something like: Can't create ne appsettings.json. File already exists at current path.